examples/helia-browser-react-verified-fetch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "helia-browser-react-verified-fetch",
```

Comment:
Maybe a simpler name?

```suggestion
  "name": "helia-verified-fetch",
```

Reply:
I'd like to keep `browser` in the name. That's a very important distinction for users.

Reply:
Ok, leave `browser` in then.